### PR TITLE
Simple helm chart added

### PR DESCRIPTION
**Pull request by apourchet**
End result of helm install looks like this:
```
$ kubectl get pods
NAME                                 READY   STATUS    RESTARTS   AGE
demo-6568968d44-2bfwc                1/1     Running   0          6m
kraken-agent-x2555                   1/1     Running   3          5m
kraken-build-index-b9767cd7b-8624l   1/1     Running   0          5m
kraken-build-index-b9767cd7b-gxwxl   1/1     Running   0          5m
kraken-build-index-b9767cd7b-tvtps   1/1     Running   0          5m
kraken-origin-6cccd79f6-4bxgl        1/1     Running   0          5m
kraken-origin-6cccd79f6-lhwh4        1/1     Running   1          5m
kraken-origin-6cccd79f6-qg46w        1/1     Running   0          5m
kraken-proxy-6695ddfc9-gclhq         1/1     Running   0          5m
kraken-testfs-79db8688dc-5kg2n       1/1     Running   0          5m
kraken-tracker-6cb8b9d7fd-9h5ff      2/2     Running   0          5m
kraken-tracker-6cb8b9d7fd-9kxdl      2/2     Running   0          5m
kraken-tracker-6cb8b9d7fd-wb9fh      2/2     Running   0          5m

$ kubectl get services
NAME                    TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)        AGE
kraken-agent-registry   NodePort    10.97.253.118   <none>        80:30081/TCP   5m
kraken-build-index      ClusterIP   None            <none>        80/TCP         5m
kraken-origin           ClusterIP   None            <none>        80/TCP         5m
kraken-proxy            ClusterIP   10.99.104.243   <none>        80/TCP         5m
kraken-testfs           ClusterIP   10.99.227.32    <none>        80/TCP         5m
kraken-tracker          ClusterIP   None            <none>        80/TCP         5m
```

### CODE REVIEW

File: helm/templates/agents.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          mountPath: /etc/config
+      volumes:
+      - name: config
+        configMap:
+          name: kraken
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: kraken-agent-registry
```
Comment (evelynl94) [inline helm/templates/agents.yaml]: remove `-registry`? agent is a registry

File: helm/values.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+num_trackers: 3
```
Comment (btai24) [inline helm/values.yaml]: helm values variables should be camel case

File: helm/templates/origins.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  template:
+    metadata:
+      labels:
+        kraken: 'true'
+        kraken-app: origin
+        kraken-peer: 'true'
+    spec:
+      containers:
+      - name: main
+        image: gcr.io/uber-container-tools/kraken-origin:v0.1.1
```
Comment (btai24) [inline helm/templates/origins.yaml]: i think its best practice to interpolate the image version

File: helm/values.yaml (hunk 2 above)
Comment (btai24) [reply helm/values.yaml]: you might find it cleaner to split out the values config into trackers, build_index, origins, etc. 

i.e:
```
trackers:
  replicas: 3

build_index:
  replicas: 3

origins:
  replicas: 3
```

see: https://github.com/helm/charts/blob/master/incubator/istio/values.yaml

Comment (apourchet) [reply helm/values.yaml]: 👍

File: helm/templates/origins.yaml (hunk 3 above)
Comment (apourchet) [reply helm/templates/origins.yaml]: Done.

**Pull request merged**